client/app/common/hero/hero.js

```diff
@@ -6,6 +6,8 @@ let heroModule = angular.module('hero', [
   uiRouter
 ])
 
-.component('hero', heroComponent);
+.component('hero', heroComponent)
+  
+.name;
 
 export default heroModule;
```

Comment:
In my projects I use this:

```javascript

export default angular
    .module('calendar', [
    ])
    // ...
    .name;

```

But I'm not sure. But maybe we should just get closer to styleguide.